Online/Matchmaking/UdpPeer.cs

```diff
@@ -569,22 +569,24 @@ public UDPPeerManager() {
             try {
                 this.socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, ProtocolType.Udp);
                 this.socket.EnableBroadcast = true;
-
                 port = DEFAULT_PORT;
-                var activeUdpListeners = IPGlobalProperties.GetIPGlobalProperties().GetActiveUdpListeners();
-                bool alreadyinuse = false;
-                for (int i = 0; i < FIND_PORT_ATTEMPTS; i++) {
-                    port = DEFAULT_PORT + i;
-                    alreadyinuse = activeUdpListeners.Any(p => p.Port == port);
-                    if (!alreadyinuse)
-                        break;
-                }
-
-                if (alreadyinuse) {
-                    throw new Exception("Failed to claim a socket port");
+                // Proton 8.0/Wine for FreeBSD bug: GetActiveUdpListeners is unavailable and not correctly emulated
+                try {
+                    var activeUdpListeners = IPGlobalProperties.GetIPGlobalProperties().GetActiveUdpListeners();
+                    bool alreadyinuse = false;
+                    for (int i = 0; i < FIND_PORT_ATTEMPTS; i++) {
+                        port = DEFAULT_PORT + i;
+                        alreadyinuse = activeUdpListeners.Any(p => p.Port == port);
+                        if (!alreadyinuse)
+                            break;
+                    }
+    
+                    if (alreadyinuse) {
+                        throw new Exception("Failed to claim a socket port");
+                    }
+                }  catch (NullReferenceException e) {
```

Comment:
```suggestion
                }  catch (Exception e) {
```
Might as well catch everything here.

Reply:
I digress, that exception and only that exception is specific to FreeBSD. We don't need to catch everything - it may hinder other errors that may occur.
Unless your plan is also to "ignore if it fails/throws"?

Reply:
Not a big issue if it throws, since we can just attempt to bind to the default port